main.cpp

```diff
@@ -10,158 +9,236 @@ video: Chapter 2 - Part 3
 
  Create a branch named Part1
 
- Purpose:  This project will teach you how to declare variables and free functions.  
- This will be the first project where the code you write will be compiled and you will be responsible for making sure it compiles before submitting it for review.
- 
- 
- 1) Write down the names of the 6 major primitive types available in C++  here:
- 
- 
- 
- 
- 
- 
- 
- 
- 
- 
-2) for each primitive type, write out 3 variable declarations inside the variableDeclaration() function on line 59.
-    a) give each variable declaration an initial value
-        - just ignore wchar_t. you do not need to declare 3 variables of type 'wchar_t'
-        - 'void' is a return type. you do not need to declare 3 variables of type 'void'.
-    b) at the end of the function, call ignoreUnused once and pass all of your variables to it. see line 71 for an example
- 
+ Purpose:  This project will teach you how to declare variables and free
+functions. This will be the first project where the code you write will be
+compiled and you will be responsible for making sure it compiles before
+submitting it for review.
+
+
+ 1) Write down the names of the 6 major primitive types available in C++  here:
+
+    int
+    float
+    double
+    bool
+    char
+    void
+    wchar_t
+
+2) for each primitive type, write out 3 variable declarations inside the
+variableDeclaration() function on line 59. a) give each variable declaration an
+initial value
+        - just ignore wchar_t. you do not need to declare 3 variables of type
+'wchar_t'

```

Comment:
The mark of a great programmer is one who does not miss the details.
You have written down seven types, the instructions asked for six.
- [ ] Please pay closer attention to the details

Reply:
This comment is not very encouraging. Luckily I'm old and pretty confident in who I am and what I'm good at. Programming has never been my strongest part as a sound designer. But I strive to be better.

With that said, I totally agree on what you're saying. I just would have put it a bit differently. Also I'm not natively speaking/writing English so excuse my language at times.  

I will definitely pay more attention to details going forward and I know by experience that's how you either avoid issues or at least solve issues.